server/helpers/is-download-disabled.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+module.exports = exports  = (item, user) => {
+	return [
+		item.type === 'package',
+		item.notAvailable === true,
+		item.canBeSyndicated === 'verify',
+		item.canBeSyndicated === 'withContributorPayment' && user.contributor_content !== true,
+		item.canBeSyndicated === 'no',
+		!item.canBeSyndicated,
+		item.canDownload < 1
```

Comment:
A thought - you'll never get this number lower than 0, so why can't this be `item.canDownload === 0`

Reply:
That should be correct, however, I left it this way to be consistent with other syndication repos
https://github.com/Financial-Times/n-syndication/blob/master/src/js/modal-download.js#L125